src/lazy/reader.rs

```diff
@@ -287,20 +286,18 @@ mod tests {
     fn expand_macro_test(
         macro_source: &str,
         encode_macro_fn: impl FnOnce(MacroAddress) -> Vec<u8>,
-        test_fn: impl FnOnce(Reader<AnyEncoding, &[u8]>) -> IonResult<()>,
+        test_fn: impl FnOnce(Reader<v1_1::Binary, &[u8]>) -> IonResult<()>,
```

Comment:
🪧 This test was written before encoding directives were implemented. It manually registers a macro definition with the reader instead. Now that 1.0 readers don't have macros, it needs to be limited to working with 1.1 data.